recordium/config.py

```diff
@@ -53,11 +53,14 @@ def __getattr__(self, key):
 
     def __setattr__(self, key, value):
         if key in self._config_options:
-            if self.data.get(key, self._config_options[key]) != value:
+            if self.data.get(key) != value:
```

Comment:
You'll need to change something here, I just noticed.

You can't use the .get(), otherwise you're forbidding me to store a config var in None.

Example, let's say that the config has a key called "foo", with default in "bar". So far, it's empty. I want to store None in "foo", the code will do `self.data.get(key) != value`, being `self.data.get(key)` equal None (because the key is not there), and `value` is None, so it won't store my value (and in the future it will use the default, "bar").

So, in this case you actually need to check "if key in self.data"... if yes, check the value, if not, store it.